driver-benchmarks/build.gradle

```diff
@@ -30,6 +30,7 @@ sourceSets {
 }
 
 dependencies {
+    compile project(path: ':bson', configuration: 'archives')
```

Comment:
Can this be reverted too?

Reply:
It is needed to pick up the version of the imported `bson` packages.